cmake/modules/FindMaya.cmake

```diff
@@ -223,11 +223,21 @@ endif()
 # all listed variables are TRUE
 include(FindPackageHandleStandardArgs)
 
-find_package_handle_standard_args(Maya
-    REQUIRED_VARS
-        MAYA_EXECUTABLE
-        MAYA_INCLUDE_DIRS
-        MAYA_LIBRARIES
-    VERSION_VAR
-        MAYA_API_VERSION
-)
+if (${PXR_MAYA_API_ONLY} AND NOT ${PXR_MAYA_API_ONLY})
```

Comment:
As I understand its the difference between

#define A

and

#define A 1

So it tests the variable exists and its not been set to true.